### PR TITLE
abortions - no longer illegal!

### PR DESCRIPTION
**Pull request by dannno**
:cl: Dannno
add: Sec hailers can now be emagged for a more rational, calm message.
/:cl:

how mad are you kor

same thing as before but now the file is legal, it's just beepsky

### CODE REVIEW

**Comment by Razharas** [comment]
Absolutely horrid its a murder

**Comment by dannno** [comment]
Did you fucking edit my changelog to correct the grammar you cunt

**Comment by AdamElTablawy** [comment]
this should be an ordinary phrase tbqh

File: code/modules/clothing/masks/hailer.dm [hunk 1]
```diff
@@ -17,6 +17,7 @@
 	var/cooldown_special
 	var/recent_uses = 0
 	var/broken_hailer = 0
+	var/safety = 1
```
Comment (KorPhaeron) [inline code/modules/clothing/masks/hailer.dm]: TRUE and FALSE instead of 1 and 0

Comment (dannno) [inline code/modules/clothing/masks/hailer.dm]: is that how it's done now or something
everything seems to be 1 and 0

Comment (KorPhaeron) [inline code/modules/clothing/masks/hailer.dm]: yeah

**Comment by dannno** [comment]
done

**Pull request merged**